src/libmr_commands.c

```diff
@@ -179,7 +186,7 @@ int TSDB_mget_RG(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
         return REDISMODULE_OK;
     }
 
-    RedisModuleBlockedClient *bc = RedisModule_BlockClient(ctx, NULL, NULL, NULL, 0);
+    RedisModuleBlockedClient *bc = RedisModule_BlockClient(ctx, NULL, NULL, &rts_free_rctx, 0);
```

Comment:
```suggestion
    RedisModuleBlockedClient *bc = RedisModule_BlockClient(ctx, NULL, NULL, rts_free_rctx, 0);
```

Reply:
@MeirShpilraien it a cosmetic change right?